### DISCUSSION TITLE
Short rumble when switching between N64 controller pak and rumble pak

### DISCUSSION THREAD

**Discussion by StupidCoder** [darthcloud/BlueRetro#162] [Ideas]
I'm using a PS5 Dual Sense controller with my N64, thanks to the amazing BlueRetro project. Thanks a lot for creating this! :)
I have only one slight problem with the setup: When using both the controller pak and rumble pak, it is hard to tell which one is currently active. When switching from controller pak to rumble pak, it would be great if the controller briefly rumbled as an indication.

**Comment by darthcloud** [comment]
Yes that would be nice, will probably do so when I work on the other rumble issue in the future.
